onnxruntime/contrib_ops/rocm/bert/attention.cu

```diff
@@ -83,9 +83,11 @@ Status Attention<T>::ComputeInternal(OpKernelContext* context) const {
   using HipT = typename ToHipType<T>::MappedType;
   using QkvProjectGeneric = GemmPermuteGenericPipeline<HipT>;
   using AttentionGeneric = GemmSoftmaxGemmPermuteGenericPipeline<HipT>;
+  using AttentionTunableOp = GemmSoftmaxGemmPermuteTunableOp<HipT>;
 
   size_t qkv_project_output_bytes = QkvProjectGeneric::GetOutputNumBytes(&attn);
-  size_t attention_workspace_bytes = AttentionGeneric::GetWorkspaceNumBytes(&attn);
+  size_t attention_workspace_bytes = std::max(AttentionGeneric::GetWorkspaceNumBytes(&attn),
```

Comment:
Check whether tuning is enabled? If not enabled, we shall not allocate space for tuning.